### PR TITLE
Scale down uploaded image before detecting faces

### PR DESCRIPTION
**Pull request by Aurintas**

### CODE REVIEW

File: WhosThat/WhosThat/BusinessLogic/PersonRecognition.cs [hunk 1]
```diff
@@ -232,9 +232,18 @@ public Bitmap FrameGrabber(object sender, EventArgs e)
         }
 
         public Bitmap FindFacesInPhoto(Image image)
-        {  
-            Image<Bgr, byte> rgbImage = new Image<Bgr, byte>(new Bitmap(image));
-            Image<Gray, byte> grayPhoto = rgbImage.Convert<Gray, byte>();
+        {
+            var maxEdgeSize = int.Parse(ConfigurationManager.AppSettings["imageWidth"]);
```
Comment (dovydasjaru) [inline WhosThat/WhosThat/BusinessLogic/PersonRecognition.cs]: This whole thing looks good, but what do you think about creating and using a separate App.config value ?

Comment (Aurintas) [reply WhosThat/WhosThat/BusinessLogic/PersonRecognition.cs]: Good idea. I was also not sure about the usage of imageWIdth and imageHeight, so used them for now. My question was why are these dimensions not equal (square shape), if we have square faces and square webcam live recognition?

Comment (LinasEin) [reply WhosThat/WhosThat/BusinessLogic/PersonRecognition.cs]: Not sure, but perhaps it is due to the better image quality?

Comment (Aurintas) [reply WhosThat/WhosThat/BusinessLogic/PersonRecognition.cs]: But how having inequal sides affects image quality?

Comment (LinasEin) [reply WhosThat/WhosThat/BusinessLogic/PersonRecognition.cs]: Perhaps not all webcams produce square video? :D

**Pull request merged**